packages/webamp-modern-2/src/skin/makiClasses/Status.ts

```diff
@@ -1,17 +1,74 @@
 import GuiObj from "./GuiObj";
+import UI_ROOT from "../../UIRoot";
 
 // Maybe this?
 // http://wiki.winamp.com/wiki/XML_GUI_Objects#.3CWasabi:StandardFrame:Status.2F.3E
 export default class Status extends GuiObj {
   static GUID = "0f08c9404b23af39c4b8f38059bb7e8f";
-  setXmlAttr(key: string, value: string): boolean {
+  _stopBitmap: string;
+  _playBitmap: string;
+  _pauseBitmap: string;
+  setXmlAttr(_key: string, value: string): boolean {
+    const key = _key.toLowerCase();
     if (super.setXmlAttr(key, value)) {
       return true;
     }
     switch (key) {
+      case "stopBitmap":
```

Comment:
I think `key` should be lower case here (see line 12), so we probably want these to be lower case as well.

Reply:
good catch! just got that working